src/Http/Cookie/Cookie.php

```diff
@@ -510,7 +510,7 @@ public function withNeverExpire()
     public function withExpired()
     {
         $new = clone $this;
-        $new->expiresAt = Chronos::parse('-1 year');
+        $new->expiresAt = Chronos::createFromTimestamp(1);
```

Comment:
For me, it's not a worse option than `-1 year` (and looks like the safest and most consistent) and is now the same as how the CAKEPHP-cookie gets expired.